eth-contracts/migrations/3_governance_migration.js

```diff
@@ -20,6 +20,9 @@ const MaxInProgressProposals = 100
 
 const MaxDescriptionLengthBytes = 250
 
+// 24hr * 60min/hr * 60sec/min / ~15 sec/block = 5760 blocks
+const ExecutionDelayBlocks = 5760
```

Comment:
We should do 13s to be consistent with other parts of the system

Reply:
same with the votingPeriod above

Reply:
fixed